infra/gcp/monitoring/k8s.io/00-provider.tf

```diff
@@ -0,0 +1,23 @@
+
+/*
+This file defines:
+- Required provider versions
+- Storage backend details
+*/
+
+terraform {
+
+  backend "gcs" {
+    bucket = "k8s-infra-clusters-terraform"
```

Comment:
@rikatz the terraform states are now in different buckets : https://github.com/kubernetes/k8s.io/pull/1952
Feel free to add a new one.